azurerm/internal/services/containers/container_registry_resource.go

```diff
@@ -56,10 +56,9 @@ func resourceContainerRegistry() *schema.Resource {
 			"sku": {
 				Type:             schema.TypeString,
 				Optional:         true,
-				Default:          string(containerregistry.Classic),
+				Default:          string(containerregistry.Basic),
 				DiffSuppressFunc: suppress.CaseDifference,
 				ValidateFunc: validation.StringInSlice([]string{
-					string(containerregistry.Classic),
```

Comment:
this is probably fine if puts where failing, but what if someone had an old one that never udpated?